flutter/lib/ui/history/result_list_item.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+
+import 'package:mlperfbench_common/data/extended_result.dart';
+import 'package:mlperfbench_common/data/results/benchmark_result.dart';
+
+import 'package:mlperfbench/localizations/app_localizations.dart';
+import 'package:mlperfbench/ui/history/utils.dart';
+import 'list_item.dart';
```

Comment:
Please use the same import style (package in this case). Please also double-check other files in this PR for the same import style.

Reply:
docker/lint fixed that

Reply:
What I mean is that `import 'list_item.dart';` should be`import 'package:mlperfbench/ui/history/list_item.dart';`. But it's ok for now. I will change it and add a lint rule for this later.